python/tests/test_udf.py

```diff
@@ -193,16 +193,16 @@ def test_local(self):
 pub fn nt2nt(i: Time) -> Result<Time, Box<dyn std::error::Error>> {
     Ok(i)
 }
-pub fn arr2arr(i: Option<Vec<Option<i32>>>) -> Result<Option<Vec<Option<i32>>>, Box<dyn std::error::Error>> {
+pub fn arr2arr(i: Option<Array<Option<i32>>>) -> Result<Option<Array<Option<i32>>>, Box<dyn std::error::Error>> {
```

Comment:
how come this test didn't fail before?

Reply:
There was a typo in the Earthfile, making python SDK tests (tests other than aggregate tests) not run.